libraries/AP_BattMonitor/AP_BattMonitor_Backend.h

```diff
@@ -71,11 +94,18 @@ class AP_BattMonitor_Backend
     // logging functions 
     void Log_Write_BAT(const uint8_t instance, const uint64_t time_us) const;
     void Log_Write_BCL(const uint8_t instance, const uint64_t time_us) const;
+    void Log_Write_BATI(uint8_t instance);
+
+    void send_mavlink_smart_battery_info(const uint8_t instance, const mavlink_channel_t chan);
 
 protected:
     AP_BattMonitor                      &_mon;      // reference to front-end
     AP_BattMonitor::BattMonitor_State   &_state;    // reference to this instances state (held in the front-end)
     AP_BattMonitor_Params               &_params;   // reference to this instances parameters (held in the front-end)
+    bool                                _log_smart_batt_info;       // set flag to log smart battery info message
+    bool                                _mavlink_smart_batt_info;   // set flag to send mavlink smart battery info message
+    uint8_t                             _instance;  // instance number
```

Comment:
This already has a meaning and a role in the Sum backend, given that this directly assigns it sometimes, we should probably use a different name here (or cope without it, which would be my personal preference).

Reply:
Removed instance